BLSQ/iaso
# [IA-3701] Fixed Django warning

Fixed Django warning about `PotentialPayment`

Related JIRA tickets : IA-3701

## Self proofreading checklist


## Doc

/

## Changes

Replaced, as suggested by Django, a `.ForeignKey(unique=True)` by a `OneToOneField()` relation.


## How to test

Start the backend, make sure that there is no more warning (or check in the github actions below :arrow_down:) 

## Print screen / video
![image](https://github.com/user-attachments/assets/00fd0ab2-5045-4872-9a91-9335b44a9ed6)


## Notes
Tests are running fine, but I'm not sure about how to thoroughly test this feature, as I don't know the business use case behind this Model.


## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
